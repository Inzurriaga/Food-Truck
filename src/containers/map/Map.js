import React, { Component } from "react";
import { Map, Marker, Popup, TileLayer } from "react-leaflet";

export default class MapContainer extends Component{
    render(){
        const position = [39.665905, -104.992843]
        const positionone = [39.7, -104.992843]
        return(
            <Map className="map" center={position} zoom="15" zoomOffset={this.zoomOffset}>
                <TileLayer
                        attribution='<a href="https://wikimediafoundation.org/wiki/Maps_Terms_of_Use">Wikimedia</a>'
                        url='https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}{r}.png'/>
                <Marker position={position}>
                    <Popup>
                        A pretty CSS3 popup.
                        <br/>
                        Easily customizable.
                    </Popup>
                </Marker>
                <Marker position={positionone}>
                    <Popup>
                        A pretty CSS3 popup.
                        <br/>
                        Easily customizable.
                    </Popup>
                </Marker>
            </Map>
        )
    }
} 