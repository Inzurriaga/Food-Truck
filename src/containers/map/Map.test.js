import React from "react";
import { shallow } from "enzyme";
import { MapContainer } from "./Map"

const mockTrucks = []
const mockPosition = []

describe("Map", () => {
    describe("Map", () => {
        let wrapper;
        beforeEach(() => {
            wrapper = shallow(
                <MapContainer trucks={mockTrucks}
                                position={mockPosition}/>
            )
        })
        it("should match snap shot", () => {
            expect(wrapper).toMatchSnapshot()
        })
    })
})