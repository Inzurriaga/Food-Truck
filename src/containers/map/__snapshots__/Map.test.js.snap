// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Map should match snap shot 1`] = `
<Map
  center={Array []}
  className="map"
  zoom="15"
>
  <ForwardRef(Leaflet(TileLayer))
    attribution="<a href=\\"https://wikimediafoundation.org/wiki/Maps_Terms_of_Use\\">Wikimedia</a>"
    url="https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}{r}.png"
  />
  <ForwardRef(Leaflet(Marker))
    position={Array []}
  >
    <ForwardRef(Leaflet(Popup))>
      your current position
    </ForwardRef(Leaflet(Popup))>
  </ForwardRef(Leaflet(Marker))>
</Map>
`;
